SU-HSDO/suhumsci
# STN-714: Fix npm start task

# READY FOR REVIEW

## Summary
This branch fixes the `npm start` task so that it runs the entire set of build tasks and watches the codebase. This should resolve differences caused when one person runs `npm start` and another runs `npm run build`.

## Need Review By (Date)
asap

## Urgency
low

## Steps to Test
1. Delete `node_modules` directory
2. Run `npm i`
2. Without editing any files run `npm run build` and confirm that there are be no changes made to compiled CSS or JS artifacts. The watch task should not start.
1. Without editing any files run `npm start` and confirm that there are be no changes made to compiled CSS or JS artifacts. The watch task should start.


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
